### ISSUE TITLE
Resolver Error when resolving existing VBA project for the first time

### ISSUE THREAD

**Issue by lucapallotta** [rubberduck-vba/Rubberduck#6189]
Thank you in advance for the help - have included details below as well as the error log.

**Rubberduck version information**
Version 2.5.9.6316
OS: Microsoft Windows NT 10.0.19045.0, x64
Host Product: Microsoft Office x64
Host Version: 16.0.17029.20068
Host Executable: POWERPNT.EXE

**Description**
Resolver Error when resolving code for first time

**Logfile**
See attached file

**Additional context**
Running resolver for the first time on a large pre-existing VBA project
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/13704575/RubberduckLog.txt)

**Comment by pflugs30** [comment]
Related to #6187, by chance?

**Comment by MDoerner** [comment]
Yes, after a short look at the log, it seems to be the same issue. I will need to find some time to  implement a fix. Unfortunately, my day job is going a bit crazy right now.

I hope to get this fixed over Christmas.

**Comment by pflugs30** [comment]
> Unfortunately, my day job is going a bit crazy right now.

Same!  Thanks for all you do.  Merry Christmas!
